taipy/core/data/_data_manager.py

```diff
@@ -68,6 +70,18 @@ def _bulk_get_or_create(
             for dn_config, owner_id in dn_configs_and_owner_id
         }
 
+    @classmethod
+    def _can_create(cls, config: DataNodeConfig) -> Reasons:
+        config_id = getattr(config, "id", None) or str(config)
```

Comment:
I don't understand the way of getting the config id from a config.

Reply:
If the object is not a proper Config with id, we will need to convert it to string to print it out

Reply:
If config is not a scenario config or a data node config, it is just a wrong object. I guess we can just say the object has an incorrect type. Do we need to see if it has an id or not? 
A message like `f"Type '{config.type}' of object '{str(config)}' is not correct." should be enough.

What do you think? In any case it is a minor remark.

Reply:
I think we can keep it this way to be consistent with the is_submittable api

Reply:
All right. 👍🏻